MalcolmMark/Free-Mentors
# #168091085 missing files

#### What does this PR do?
This PR adds the necessary missing files to power the UI template for free mentors
#### Description of Task to be completed?
- add admin file into its correct directory
- add assets in the UI
#### How should this be manually tested?
This can not be manually tested however view changes after cloning this repo upon the missing files that were added and notice the different effects
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168091085
#### Screenshots (if appropriate)
- Before adding assets and missing files
![UsersCan View Mentors](https://user-images.githubusercontent.com/45765235/63629867-83d68e00-c615-11e9-807f-d8806da76416.PNG)

- After adding assets and missing files
![after adding missing files](https://user-images.githubusercontent.com/45765235/63629973-766dd380-c616-11e9-9864-bc24ffc1f649.PNG)

#### Questions:
n/a
